mlora/executor/context/context.py

```diff
@@ -11,16 +11,16 @@ class TaskContext(metaclass=ABCMeta):
     type_: str = ""
     name_: str = ""
     path_: str = ""
-
+    now_step_: int = 0
     device_: str = ""
-
+    last_epoch: int = -1
```

Comment:
what's the meaning of the last epoch?

Reply:
Make the learning rate to be same as the state before fault  which we saved

Reply:
torch.optim.lr_scheduler.CosineAnnealingLR(optimizer,T_max,eta_min=0,last_epoch=-1) can use the parameter "last_epoch" to ensure the learning rate to be same as the state before fault which we saved